ic-os/guestos/defs.bzl

```diff
@@ -98,8 +98,20 @@ def image_deps(mode, malicious = False):
 
     # Update dev rootfs
     if "dev" in mode:
+        # Allow console access
+        deps["rootfs"].update({"//ic-os/guestos/context:allow_console_root": "/etc/allow_console_root:0644"})
+
+        # Dev config tool
         deps["rootfs"].pop("//rs/ic_os/release:config", None)
         deps["rootfs"].update({"//rs/ic_os/release:config_dev": "/opt/ic/bin/config:0755"})
-        deps["rootfs"].update({"//ic-os/guestos/context:allow_console_root": "/etc/allow_console_root:0644"})
+
+    # Update recovery rootfs
```

Comment:
(Following above)

Maybe instead can we add these directly to `rootfs` like we do the others above? It would be nicer on the cache - the container steps would be shared, with only the `rootfs` and above, changing - and we could use the separation to make it more clear that these are not and should not be on every image. The extra files could live closer to `dev-recovery` with a readme, for example.